packages/oid4vci-rest-client/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@sphereon/ssi-sdk.oid4vci-rest-client",
+  "description": "contains the client side to call REST endpoints of a Verifiable Credential Issuer",
+  "version": "0.11.0",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "IOID4VCIRestClient": "./src/types/IOID4VCIRestClient.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "build:clean": "tsc --build --clean && tsc --build",
+    "generate-plugin-schema": "veramo dev generate-plugin-schema"
```

Comment:
wrong script

Reply:
changed to `ts-node ../../packages/dev/bin/sphereon.js dev generate-plugin-schema`